traitsui/tests/editors/test_datetime_editor.py

```diff
@@ -251,8 +251,8 @@ def test_datetime_editor_qt_datetime_out_of_bound(self):
                 self.launch_editor(instance, view) as editor:
             # the user set the datetime on the Qt widget to a value
             # too large for Python
-            from pyface.qt.QtCore import QDateTime, QDate
-            q_datetime = QDateTime(QDate(datetime.MAXYEAR + 1, 1, 1))
+            from pyface.qt.QtCore import QDateTime, QDate, QTime
+            q_datetime = QDateTime(QDate(datetime.MAXYEAR + 1, 1, 1), QTime())
```

Comment:
`QTime()` without any argument causes it to use the local time, which varies from location to location, run to run. While I understand the value isn't really relevant here (because the date itself is large enough such that the value is already out-of-bound no matter what time is), typically we want the tests to run in a predictable manner wherever the machine is. Could you specify the hour, minute, second and millisec to 0 please?

Reply:
I was under the impression from the docs that `QTime()` constructed a Null time object (i.e. `isNull` returns `True` and `isValid` returns `False`).  And then `QDateTime` docs on the constructor say "If date is valid and time is not, the time will be set to midnight."
I am happy to explicitly set `QTime(0,0)` as it is clearer, but I think they should have the same effect?

Reply:
Ah!  I think I was reading the output from a different test, another test is using `datetime.now()`, not this one. Sorry about that. I think you are right, and it is still better to be explicit than be implicit. It makes the test easier to reason about.

Reply:
But feel free to revert that if you'd prefer a null time... in other tests the time isn't explicitly set either.